### PR TITLE
fix: missing subscript

### PR DESCRIPTION
**Pull request by FelixKoTU**
Subscript "s" is missing for the number of cells in series (N_s) in latex math expressions.

### CODE REVIEW

File: docs/sphinx/source/user_guide/singlediode.rst [hunk 1]
```diff
@@ -111,4 +111,4 @@ Clifford W. Hansen, Sandia `Report SAND2015-2065
 
 [4] "Computer simulation of the effects of electrical mismatches in
 photovoltaic cell interconnection circuits" JW Bishop, Solar Cell (1988)
-:doi:`10.1016/0379-6787(88)90059-2`
\ No newline at end of file
+:doi:`10.1016/0379-6787(88)90059-2`
```
Comment (RDaxini) [inline docs/sphinx/source/user_guide/singlediode.rst:114]: What changed here? It looks like nothing happened, but it shows as a change

**Review by RDaxini — COMMENTED** [review]
Thanks for spotting and fixing this @FelixKoTU.
Shall we expand the scope a little bit? We could link the references properly in line 20: `by Jain and Kapoor _[1] _[2], and Hansen _[3].` We could also standardize the formatting of the reference list. See [here](https://pvlib-python.readthedocs.io/en/latest/contributing/style_guide.html#references) for further info

File: docs/sphinx/source/user_guide/singlediode.rst (hunk 1 above)
Comment (FelixKoTU) [reply docs/sphinx/source/user_guide/singlediode.rst:114]: This seems to be an oversight on my part, I did not mean to alter that line.

Comment (cwhanse) [reply docs/sphinx/source/user_guide/singlediode.rst:114]: > We could link the references properly in line 20: by Jain and Kapoor _[1] _[2], and Hansen _[3]. 
> We could also standardize the formatting of the references

Only if @FelixKoTU wants to, in my view.

This page really doesn't fit in a User Guide, it's more of detailed background we ought to have somewhere but didn't know where else to put it. My point is that we ought to consider rewriting this page and editing the references can be done in that revision.

Comment (RDaxini) [reply docs/sphinx/source/user_guide/singlediode.rst:114]: > Only if @FelixKoTU wants to, in my view.
> 
> This page really doesn't fit in a User Guide, it's more of detailed background we ought to have somewhere but didn't know where else to put it. My point is that we ought to consider rewriting this page and editing the references can be done in that revision.

Fair point!

**Review by RDaxini — APPROVED** [review]

**Comment by kandersolar** [comment]
> What changed here? It looks like nothing happened, but it shows as a change

I think it's whether the line ends in `\n` or not: https://github.com/FelixKoTU/pvlib-python/commit/5359efe3cd564807792cd99f6a80647928736859.patch

Doesn't really matter IMHO.  Thanks @FelixKoTU!

**Comment by RDaxini** [comment]
> Doesn't really matter IMHO

for sure no big deal I was kinda just curious 😅

**Pull request merged**